From 4700d836e0caa529de321618f9b94670c1960ac3 Mon Sep 17 00:00:00 2001
From: Rui Loura <rui@joyent.com>
Date: Mon, 11 Dec 2017 15:49:04 +0000
Subject: [PATCH] HEAD-2382 Dead link in README.md Reviewed by: Cody Mello
 <cody.mello@joyent.com> Approved by: Cody Mello <cody.mello@joyent.com>

---
 README.md | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/README.md b/README.md
index 8d14cd08..884a6908 100644
--- a/README.md
+++ b/README.md
@@ -44,7 +44,7 @@ To create a VM for local development work – commonly called 'coal' (Cloud On
 
   - open `coal-master-TIMESTAMP-gSHA.vmwarevm`, select 'Live 64-bit' at the
     grub menu, and work through the interactive installer referring to [this
-    documentation](). **Important**: while many answers are arbitrary, the
+    documentation][coal-setup.md]. **Important**: while many answers are arbitrary, the
     networking questions require specific values for local development.
 
   - when setup completes, you can access the headnode via ssh: `ssh
@@ -454,3 +454,4 @@ To test changes to setup procedures without a complete rebuild, you can:
 [manta]: https://github.com/joyent/manta
 [buildspec]: #build-specification-buildspec-and-buildspeclocal
 [autosetup]: #automating-headnode-setup-answersjson
+[coal-setup.md]: https://github.com/joyent/triton/blob/master/docs/developer-guide/coal-setup.md
-- 
2.21.0

